x-pack/plugins/ml/public/application/components/model_snapshots/revert_model_snapshot_flyout/revert_model_snapshot_flyout.tsx

```diff
@@ -0,0 +1,396 @@
+            height={'100px'}
+            width={'100%'}
+            fadeChart={true}
+            overlayRanges={[
+              {
+                start: currentSnapshot.latest_record_time_stamp,
+                end: job.data_counts.latest_record_timestamp,
+                color: '#ff0000',
+              },
+            ]}
+          />
+
+          <EuiSpacer size="l" />
+          <EuiSpacer size="l" />
+
+          <EuiCallOut
+            title={i18n.translate(
+              'xpack.ml.newJob.wizard.revertModelSnapshotFlyout.warningCallout.title',
+              {
+                defaultMessage: 'Anomaly data will be deleted',
+              }
+            )}
+            color="warning"
+            iconType="alert"
+          >
+            <FormattedMessage
+              id="xpack.ml.newJob.wizard.revertModelSnapshotFlyout.warningCallout.contents"
+              defaultMessage="All anomaly detection results after {date} will be deleted."
+              values={{ date: formatDate(currentSnapshot.latest_record_time_stamp, TIME_FORMAT) }}
+            />
+          </EuiCallOut>
+
+          <EuiHorizontalRule margin="xl" />
+
+          <EuiFormRow
+            fullWidth
+            helpText="Reopen job and replay analysis after the revert has been applied."
+          >
+            <EuiSwitch
+              id="replaySwitch"
+              label={i18n.translate(
+                'xpack.ml.newJob.wizard.revertModelSnapshotFlyout.replaySwitchLabel',
+                {
+                  defaultMessage: 'Replay analysis',
+                }
+              )}
+              checked={replay}
+              onChange={(e) => setReplay(e.target.checked)}
+            />
+          </EuiFormRow>
+
+          {replay && (
+            <>
+              <EuiFormRow
+                fullWidth
+                helpText={
+                  runInRealTime
+                    ? 'Job will continue to run until manually stopped. All new data added to the index will be analyzed.'
+                    : `Job will run for the same length of time as it is currently (${formatDate(
+                        job.data_counts.latest_record_timestamp,
+                        TIME_FORMAT
+                      )}). It will then stop and close.`
+                }
+              >
+                <EuiSwitch
+                  id="realTimeSwitch"
+                  label={i18n.translate(
+                    'xpack.ml.newJob.wizard.revertModelSnapshotFlyout.realTimeSwitchLabel',
+                    {
+                      defaultMessage: 'Run job in real time',
+                    }
+                  )}
+                  checked={runInRealTime}
+                  onChange={(e) => setRunInRealTime(e.target.checked)}
+                />
+              </EuiFormRow>
+
+              <EuiFormRow
+                fullWidth
+                helpText="Create a new calendar and event to skip over a period of time when analyzing the data."
```

Comment:
More text for i18n. And looks like you might need to remove the commented out line below.